README.md

```diff
@@ -51,19 +51,19 @@ The `main` branch always contains latest code, so better use some package manage
 	- all enums, described in section 5, must be typed with section name singularized (e.g., "5.2 Banner Ad Types" -> `type BannerAdType int8`)
 	- all typed enums must have constants for each element, prefixed with type name (e.g., "5.2 Banner Ad Types - XHTML Text Ad (usually mobile)" -> `const BannerAdTypeXHTMLTextAd BannerAdType = 1`)
 	- never use `iota` for enum constants
-	- OpenRTB (2.x) section "5.1 Content Categories" should remain untyped and have no constants
+	- OpenRTB (2.x) "content categories" should remain untyped and have no constants
```

Comment:
This section no longer exists but there are still several references to content categories.